create.js

```diff
@@ -83,19 +83,18 @@ module.exports = function create (path, opts, keys) {
       return db.keys.get(key, function (err, data) {
         if (err) return cb(err)
 
-        if (isPrivate && unbox) {
-          data = db.unbox(data, unbox)
+        if (!isPrivate) {
+          if (meta) cb(null, { key, value: data.value, timestamp: data.timestamp })
+          else cb(null, data.value)
         }
+        else {
+          db.unbox(data, unbox, function (err, result) {
+            if (err) return cb(err)
 
-        let result
-
-        if (isPrivate) {
-          result = data.value
-        } else {
-          result = u.originalValue(data.value)
+            if (meta) cb(null, { key, value: result.value, timestamp: result.timestamp })
+            else cb(null, result.value)
+          })
```

Comment:
Cool! Looks like we're only unboxing when we need to, which is great. Have you ran benchmarks on this code? It's important that it doesn't slow anything down, because this is a very hot path, but I have a hunch that this might even speed up methods like `createHistoryStream()`.

Reply:
From call

use https://github.com/ssbc/bench-ssb

run 1-6
---

run these benches before + after on my system

Reply:
## Process

- clone down [`bench-ssb`](https://github.com/ssbc/bench-ssb), npm linking `ssb-db`
- ran `run-all.sh` ... saw output turn up, but bench didn't get beyond test 9 for some reason (so I quit out)
- changed branch of `ssb-db` (back to master)
- changed `run-all.sh` to 
  - skip generate (so using same output.json as first test)
  - append `>>` results, instead of overwrite `>` (so can easily compare)
  - skipped tests `>= 9`

## Results

here's a sampling of results from tests 1-8
```
// 04-out.txt
minimal 100001 1 7197.941409342834 13.893 // async-box branch
minimal 100001 1 6958.527590285992 14.371 // master branch
```
```
07-out.txt
clock-dump 99001 9001 14375.054450413823 6.887  // async-box branch
clock-dump 99001 9001 14629.968967045957 6.767  // master branch
```

```
// 08-out.txt
sbot-read 100002 2 10718.327974276528 9.33  // async-box branch
sbot-read 100002 2 9706.105017955935 10.303 // master branch
```